processes/executive-copy-creation-checklist.md

```diff
@@ -0,0 +1,57 @@
+
+# Executive Creation
+
+## Creation
+
+- [ ] Sync Community Git
+- [ ] Ensure you are working from the latest master branch
+- [ ] Read Associated Executive Contents sheet
+- [ ] Copy the executive template
+- [ ] Update filename
+- [ ] Update dates - exact time is not required and should be left at midnight.
+
+- [ ] Ensure the title refers to the most impactful/important actions included in the proposal.
+- [ ] Ensure that the summary refers to all actions included in the proposal.
+
+- [ ] Ensure that the publisher of the executive is listed in the preamble (at the entity level).
+- [ ] Ensure that the writer of the executive is listed in the preamble (at the entity level).
+- [ ] Ensure that the reviewers of the executive are listed in the preamble (at the entity level).
+
+- [ ] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
+- [ ] Ensure that it is clearly stated which items are affected by the GSM Delay.
+- [ ] Ensure that the GSM Delay period is clearly communicated.
+- [ ] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.
+
+## Items
+
+- [ ] Add each item to the proposal details section
+- [ ] Order the items by judged impact/importance
```

Comment:
per crafters request - try and make sure that the order in the sheet and the order in the copy match to aid code review